services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -135,6 +134,11 @@ public Response getMainPage() throws IOException, FreeMarkerException {
             }
         });
 
+        String devServerUrl = System.getenv("KEYCLOAK_ACCOUNT_VITE_URL");
```

Comment:
I'll rename the constants to match the existing conventions. But I'd rather keep this outside of the `ViteManifest`, as it has a single responsibility to represent a manifest, and should not concern itself with where that data is coming from.
 
The main issue is that we have no common class for themes that they can extend from, ideally the manifest file extraction functionality (and the constant for the dev URL) are all part of a base class that various themes can just extend, but this would be a larger refactor effort.

Reply:
For me it is still related to the manifest (and SoC) so it should be fine to have the constant there and not anywhere else ...

Reply:
I can add it there if you really feel so strongly about it, but I still have to disagree.